terraso_backend/apps/soilproj/models.py

```diff
@@ -0,0 +1,73 @@
+# Copyright © 2021-2023 Technology Matters
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see https://www.gnu.org/licenses/.
+from django.db import models
+
+from apps.core import permission_rules
+from apps.core.models import User
+from apps.core.models.commons import BaseModel, SlugModel
+
+
+class Project(BaseModel):
+    class Meta(BaseModel.Meta):
+        abstract = False
+
+        rules_permissions = {"add_site": permission_rules.can_add_site}
+
+    PRIVATE = "pri"
+    PUBLIC = "pub"
+    PRIVACY_OPTIONS = [(PRIVATE, "Private"), (PUBLIC, "Public")]
+
+    name = models.CharField(max_length=200)
+    privacy = models.CharField(max_length=3, choices=PRIVACY_OPTIONS, default=PRIVATE)
+    members = models.ManyToManyField(User, through="ProjectMembership")
```

Comment:
question: there was discussion of having the project contain a group in the way that landscapes do to manage membership, do you have thoughts on this? i'm not sure what the pros and cons would be. like the vague pro would be that it might make integrating LandPKS with Terraso's other features easier, & the vague con would be that it would couple projects to groups in a potentially annoying way in the future.

Reply:
Interesting thought! I am not against this but it would make sense to think it out a bit further before we try implementing this.

Reply:
I agree with @shrouxm that is was the expectation since the group has all the logic to handle memberships, the idea was to implement this similarly to landscape

Reply:
I think you're right @josebui , we need to think through how these concepts will interact. I'm working on an architecture issue for this right now, I'll try adding my thoughts/questions there.